src/dex-ui/pages/dao/DashboardHeader.tsx

```diff
@@ -20,11 +22,22 @@ interface DashboardHeaderProps {
   name: string;
   type: DAOType;
   logoUrl?: string;
+  isNFT?: boolean;
+  handleMintNFT?: (tokenLinks: string[]) => void;
 }
 
 export function DashboardHeader(props: DashboardHeaderProps) {
-  const { daoAccountId, name, type, logoUrl, govTokenId, safeId, isMember, isAdmin } = props;
+  const { daoAccountId, name, type, logoUrl, govTokenId, safeId, isMember, isAdmin, isNFT, handleMintNFT } = props;
   const navigate = useNavigate();
+  const { data: token } = useToken(govTokenId ?? "");
+  let showMintNFTButton = false;
+  if (isNFT && token) {
+    const maxSupply = token.data.max_supply;
+    const totalSupply = token.data.total_supply;
+    if (maxSupply && totalSupply) {
+      showMintNFTButton = Number(token.data.max_supply) > Number(token.data.total_supply);
+    }
+  }
 
```

Comment:
we can simplify the above condition:

` const { daoAccountId, name, type, logoUrl, govTokenId = "", safeId, isMember, isAdmin, isNFT, handleMintNFT } = props;
  const navigate = useNavigate();
  const { data: token } = useToken(govTokenId);
  const showMintNFTButton = type === DAOType.NFT && Number(token?.data.max_supply) > Number(token?.data.total_supply);`

Reply:
You are right. made the suggested changes.